src/dependency-manager/manager.ts

```diff
@@ -319,13 +338,25 @@ export default abstract class DependencyManager {
           environment: {},
         };
       }
+
       const service_ref = buildNodeRef(component_config, 'services', service_name);
       for (const [interface_name, interface_config] of Object.entries(service.interfaces)) {
         const interface_ref = `services.${service_name}.interfaces.${interface_name}`;
 
         const architect_host = service_ref;
         const architect_port = `${interface_ref}.external_port`;
 
+        // TODO: Remove once databases get their own node type
```

Comment:
It would only really be necessary when we eventually add provisioning of managed DBs (e.g. RDS) so that we know to fulfill the node differently. Since this is just transforming it into a service its not necessary.